code/game/machinery/doors/windowdoor.dm

```diff
@@ -103,18 +103,18 @@
 	icon_state = "[src.base_state]open"
 	flick("[src.base_state]opening", src)
 	playsound(src.loc, 'sound/machines/windowdoor.ogg', 100, 1)
-	addtimer(CALLBACK(src, .proc/open_final), 10, TIMER_UNIQUE | TIMER_OVERRIDE)
+	addtimer(CALLBACK(src, .proc/open_final), 0.9 SECONDS, TIMER_UNIQUE | TIMER_OVERRIDE)
```

Comment:
Why do we override here but not in the other places?

Reply:
I am not sure we need timer override. The logic of this proc won't let `addtimer` while the door is `operating`,
so it's unreachable, until the timer end.